docker-compose.yml

```diff
@@ -68,7 +68,7 @@ services:
     ports:
       - "127.0.0.1:8889:8888"
     volumes:
-      - ./notebooks:/notebooks
+      - "${NOTEBOOKS:-./notebooks}:/notebooks"
```

Comment:
After discussing the complexity this would introduce in `script/test`, I think it's better to just drop it.